presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientModule.java

```diff
@@ -111,6 +114,13 @@ public static CassandraSession createCassandraSession(
 
         LoadBalancingPolicy loadPolicy = new RoundRobinPolicy();
 
+        if (config.getUseTls()) {
```

Comment:
Hmm, I think it's reasonable to use the default stores if you don't specify a custom keystore/truststore, we don't need a new property